wagtail/images/static_src/wagtailimages/scss/chooser-duplicate-upload.scss

```diff
@@ -0,0 +1,21 @@
+.duplicate-upload {
+  &__figure {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    img {
+      margin: 1rem auto;
+    }
+  }
+
+  &__actions {
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 2.5rem;
+
+    form {
```

Comment:
Try to avoid using direct element selectors if you can - lean towards a class, following BEM